ckanext/cfpb_extrafields/plugin.py

```diff
@@ -277,3 +292,21 @@ def update_config(self, config):
         # Add this plugin's templates dir to CKAN's extra_template_paths, so
         # that CKAN will use this plugin's custom templates.
         tk.add_template_directory(config, 'templates')
+
+    p.implements(p.IFacets)
+    def change_facets(self, facets_dict):
+        dummy_facets = facets_dict
+        facets_dict = collections.OrderedDict()
+        # example facet added
+        facets_dict['acquisition_method'] = p.toolkit._('Acquisition Method')
+        for key in dummy_facets.keys():
+            facets_dict[key] = dummy_facets[key]
```

Comment:
OrderedDicts don't support ordered insert, so inserting at position 0 (top of the page) wouldn't be possible with an update() sadly.